diff --git a/Grayjay.ClientServer/Grayjay.ClientServer.csproj b/Grayjay.ClientServer/Grayjay.ClientServer.csproj
index f1a0467..b16e232 100644
--- a/Grayjay.ClientServer/Grayjay.ClientServer.csproj
+++ b/Grayjay.ClientServer/Grayjay.ClientServer.csproj
@@ -8,7 +8,7 @@
     <OutputType>library</OutputType>
     <RuntimeIdentifiers>win-x86;win-x64;win-arm64;linux-x64;linux-arm;linux-arm64;osx-x64;osx-arm64</RuntimeIdentifiers>
     <RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
-    <AssemblyVersion>1.10.0.0</AssemblyVersion>
+    <AssemblyVersion>1.11.0.0</AssemblyVersion>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)'=='Release'">
